### DISCUSSION TITLE
New benchmarking result for janssen

### DISCUSSION THREAD

**Discussion by aguskwee** [JanssenProject/jans#10186] [Q&A] [answered]
Hi,

I saw this benchmarking page for Janssen. https://docs.jans.io/v1.1.5/admin/recipes/benchmark/

I just wondering whether you have the new benchmarking result for Janssen.

Have you ever tried other benchmarking load test such as username + password + OTP authentication flow?

Regards,
Agus

**Comment by ossdhaval** [comment]
Tagging @moabu for comments.

**Comment by moabu** [comment, marked as answer]
Hey @aguskwee ,

Yes . New results get moved to newer versions unless the results change. We haven’t added an ROPC test with OTP results . The OTP is just another step in terms of load testing. I guess we can setup a jmeter test for that. I’ll open an issue for that . 

Please follow https://github.com/JanssenProject/jans/issues/10221
